.github/build.yml

```
@@ -0,0 +1,34 @@
+name: Build Docker Image
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
```

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+"
```
It looks like our [git tags](https://github.com/ratehub/ratehub-express-gateway/tags) actually start with a "v" if I'm reading this correctly - whether we want to keep that or not is a separate discussion though (the "v" on a version is usually seen as redundant).

---

Since it is a regexp match you could make the "v" optional too.